README.md

```diff
@@ -1,4 +1,72 @@
 NGOConnect-Api-server
 =====================
 
-API server
+###Database Used: MongoDB###
+	
+#####Why we've used MongoDB?#####
+
+1. MongoDB is a database but is not in SQL family of databases, i.e. its not a relational database,
+2. MongoDB is a document oriented database, that is it stores structured data as JSON like documents with dynamic schemas. Instead of storing data in tables, which is 		what RDB do.
+3. Pass a JSON object, it stores them easily without predefining the schema(the fact that it takes JSON objects directly makes it ideal for NODEJS), this data 			 flexibility has made MongoDB so popular.
+
+
+-----------------------------------------------------------------------------------------------------------------------------------------------
+
+####Steps to setup MongoDB server:####
+
+1. Download and install your OS specific bundle from: http://mongodb.org/downloads
+2. Launch MongoDB : $ (sudo) mongod
+
+
+
+------------------------------------------------------------------------------------------------------------------------------------------------
+
+####Integration with NodeJS Server:####
+
+1. MongoJS is a very simple wrapper for MongoDB driver for NODEJS, it mimics the actual MongoDB Api very closely.
+2. Grab MongoJS : npm install mongojs
+3. Require MongoJS and assign it to an object, var db = require('mongojs');.
+4. Now, db.connect(dburl, collections); //here dburl = "localhost/databasename" //Collections: similar to tables RDB, used to group similar data together.
+
```

Comment:
Till this point ..the steps should be added to the create_database.js file in our wiki.
After this part the content is not needed.create_database is sufficiet.